extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -44,11 +43,16 @@ const buildSliderTemplate = (element) => {
             value="0"
           />
           <div class="i-amphtml-story-interactive-slider-bubble"></div>
+          <div class="i-amphtml-story-interactive-slider-average-bubble"></div>
```

Comment:
Same as other comment, we could use a more descriptive name like `average-indicator`